Assets/Scripts/SS3D/Systems/Interactions/InteractionController.cs

```diff
@@ -36,6 +37,7 @@ protected override void OnStart()
             _camera = SystemLocator.Get<CameraSystem>().PlayerCamera.GetComponent<Camera>();
         }
 
+        [Client]
```

Comment:
This is called both on client and server. It will throw warnings every frame in Server Only mode.